downloader/src/main/kotlin/WorkingTree.kt

```diff
@@ -34,7 +34,10 @@ abstract class WorkingTree(val workingDir: File, val vcsType: VcsType) {
 
     /**
      * Conveniently return all VCS information about how this working tree was created, so it could be easily
-     * recreated from that information.
+     * recreated from that information. Note that the returned path is a bit special in that regard as it always just
+     * contains the relative path of [workingDir] to [getRootPath]. That is, it is not related to the path argument that
+     * was used for downloading, and at the example of Git, it does not reflect the (single) path the was cloned in a
+     * sparse checkout.
```

Comment:
I think the "is a bit special in that regard as it always" and "That is" can be removed, it's not helping the understanding.
"**that** was cloned"
Why "(single)"?

Reply:
> Why "(single)"?

Because in plain Git a sparse checkout could be configured to match multiple paths. Just *our* VcsInfo API only supports a *single* path.